modules/kubernetes-addons/variables.tf

```diff
@@ -265,6 +265,43 @@ variable "crossplane_kubernetes_provider" {
   }
 }
 
+#-----------ISTIO ADDON------------------
+variable "enable_istio" {
```

Comment:
currently this doesn't do anything if changed to `true`. what is the "default" for installing Istio - this should be the behavior that is enabled when setting this to `true`

should base, cni, and istiod all be installed in the "default" scenario? or is only "base" required for the MVP of Istio install?